waspc/src/Generator.hs

```diff
@@ -8,23 +8,24 @@ import Data.Time.Clock
 import qualified Paths_waspc
 import qualified Data.Version
 import Control.Monad (mapM_)
-import Path ((</>), relfile)
-import qualified Path
-import qualified Path.Aliases as Path
+import qualified Path as P
 
+import StrongPath (Path, Abs, Dir, (</>))
+import qualified StrongPath as SP
 import CompileOptions (CompileOptions)
 import Wasp (Wasp)
 import Generator.WebAppGenerator (generateWebApp)
 import Generator.ServerGenerator (genServer)
 import Generator.FileDraft (FileDraft, write)
+import Generator.Common (ProjectRootDir)
```

Comment:
did not get here from name that this is root dir of "destination", where the project will be generated. Maybe we could add in into the name?

Reply:
Mmmm maybe but now I used it in lot of places already. But it is in Generator module so that should tell you a lot, plus there is comment next to it describing it. But if we have a better name we can change it. How would you call it? My idea was to call this generate thing a "project", and it has root, and then in that root are web app and server and what not.